avocado/core/job.py

```diff
@@ -462,6 +465,7 @@ def _run(self):
         """
         self._setup_job_results()
         self.__start_job_logging()
+        self.job_pre_post_dispatcher.map_methods('pre', self)
```

Comment:
sysinfo is disabled. It's a tough call and later I think we'll need a finer granularity for this (like an option to enable/disable prepost plugins, by default disable them (in dry-run), but when user asks for it, enable them.

```python
        if getattr(self.args, "dry_run", False):  # Modify args for dry-run
            if not self.args.unique_job_id:
                self.args.unique_job_id = "0" * 40
            self.args.sysinfo = False
            if self.args.logdir is None:
                self.args.logdir = tempfile.mkdtemp(prefix="avocado-dry-run-")
```

Anyway the only thing I'm waiting for is the trello card (or disagreement) to move the job heading in front of these. So what do you think?